cadc-gms/src/main/java/org/opencadc/auth/StandardIdentityManager.java

```diff
@@ -244,18 +247,25 @@ public Subject augment(Subject subject) {
     @Override
     public Subject toSubject(Object owner) {
         Subject ret = new Subject();
+        OpenIdPrincipal p = null;
         if (owner != null) {
-            UUID uuid = null;
-            if (owner instanceof UUID) {
-                uuid = (UUID) owner;
-            } else if (owner instanceof String) {
-                String sub = (String) owner;
-                uuid = UUID.fromString(sub);
+            if (owner instanceof String) {
+                String[] openIDComponents = ((String)owner).split(OID_OWNER_DELIM);  // "issuer openID"
+                if (openIDComponents.length != 2) {
+                    throw new RuntimeException("unexpected owner format: " + owner.getClass().getName() + " value: " + owner);
```

Comment:
I wonder if we could retain backwards compatibility here by treating length==1 as a bare `sub` value, presumably from the single known oidcIssuer?? In  theory we could but it might just push the problem someplace else... we should discuss the next step(s) before deciding this and/or the version number (due to incompat).